templates/prescription/general_edit.html.twig

```diff
@@ -91,195 +89,259 @@
 </head>
 <body id="prescription_edit">
 <div class="container">
-    <form name="prescribe" id="prescribe"  method="post" action="{$FORM_ACTION}" >
+    {# TODO: @adunsulag shouldn't FORM_ACTION be escaped? #}
```

Comment:
yes, would escape it

Reply:
Removed the overall FORM_ACTION piece and broke it up into sub components so we could escape this better.